testing/tests/DevExpress.ui.widgets.dataGrid/editing.tests.js

```diff
@@ -3768,6 +3768,31 @@ QUnit.test("Insert several rows and remove they with edit mode batch", function(
     });
 });
 
+// T808395
+QUnit.test("First cell should be focused after inserting new row if startEditAction is 'dblClick'", function(assert) {
+    // arrange
+    var that = this,
+        headerPanel = this.headerPanel,
+        rowsView = this.rowsView,
+        testElement = $('#container');
+
+    that.options.editing = {
+        allowAdding: true,
+        mode: 'batch',
+        startEditAction: "dblClick"
+    };
+
+    headerPanel.render(testElement);
+    rowsView.render(testElement);
+
+    // act
+    this.addRow();
+    this.clock.tick(300);
+
+    // assert
+    assert.equal(getInputElements(testElement.find('tbody > tr').eq(0)).length, 1, 'When insert row and batch editing - focus first cell');
```

Comment:
Assert description is complex. Change it to "first row has editor"
or change assert to
assert.equal(getInputElements($(this.getCellElement(0, 0))).length, 1, "first cell has editor");